specification/monitor/resource-manager/Microsoft.Insights/preview/2020-01-01-preview/managementGroupDiagnosticSettings_API.json

```diff
@@ -0,0 +1,342 @@
+          }
+        },
+        "x-ms-examples": {
+          "Deletes the management group diagnostic setting": {
+            "$ref": "./examples/deleteManagementGroupDiagnosticSetting.json"
+          }
+        }
+      }
+    },
+    "/providers/microsoft.management/managementGroups/{managementGroupId}/providers/microsoft.insights/diagnosticSettings": {
+      "get": {
+        "tags": [
+          "ManagementGroupDiagnosticSettings"
+        ],
+        "operationId": "ManagementGroupDiagnosticSettings_List",
+        "description": "Gets the active management group diagnostic settings list for the specified management group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ManagementGroupIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to get more information about management group diagnostic setting",
+            "schema": {
+              "$ref": "#/definitions/ManagementGroupDiagnosticSettingsResourceCollection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets the management group diagnostic setting": {
+            "$ref": "./examples/listManagementGroupDiagnosticSettings.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagementGroupProxyOnlyResource": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the resource"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "A proxy only azure resource object"
+    },
+    "ManagementGroupLogSettings": {
+      "properties": {
+        "category": {
+          "type": "string",
+          "description": "Name of a Management Group Diagnostic Log category for a resource type this setting is applied to."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "a value indicating whether this log is enabled."
+        }
+      },
+      "required": [
+        "category",
+        "enabled"
+      ],
+      "description": "Part of Management Group diagnostic setting. Specifies the settings for a particular log."
+    },
+    "ManagementGroupDiagnosticSettings": {
```

Comment:
Is the diagnosticSettings definition different at MG level than other levels?  Why can't it use the same definition used for normal diagnostic settings?

Reply:
The definition is essentially the same. 
The path is different -- so a normal diagnostic setting has <resourceUri>/providers/microsoft.insights/diagnosticSettings
Management group will have "/providers/microsoft.management/managementGroups/<mgId>/providers/microsoft.insights/diagnosticSettings"

My understanding is that if I am introducing a new api-version, and a new file I would need to define this again (Please see how subscriptionDiagnosticSettings has been done-- https://github.com/Azure/azure-rest-api-specs/blob/master/specification/monitor/resource-manager/Microsoft.Insights/preview/2017-05-01-preview/subscriptionDiagnosticsSettings_API.json)